server/lib/article/assemble.js

```diff
@@ -60,6 +60,11 @@ const extraArticleData = (article, options) => promiseAllObj({
 	css: getCSS(article, options),
 }).then(extra => Object.assign(article, extra));
 
+// HACK: hide certain articles from the AMP cache
+const articlesToSkip = [
+	'263615ca-d873-11e9-8f9b-77216ebe1f17', // general election poll tracker: contains image that shouldn't be cached
```

Comment:
This is probably silly, but do you mind putting that information in the description of the PR to make it easier for whoever looks this up in the future?

Reply:
nope, good idea